FreeRTOS/FreeRTOS
# Add SMP schedule highest priority on-target test

Add SMP schedule highest priority on-target test

Description
-----------
Adding the on target test to verify that
>The scheduler shall correctly schedule the highest priority ready tasks.

Test Steps
-----------
Test result on RP2040
```
./FreeRTOS/FreeRTOS/Test/Target/tests/smp/schedule_highest_priority/schedule_highest_priority.c:215:Test_ScheduleHighestPriority:PASS

-----------------------
1 Tests 0 Failures 0 Ignored
```


Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
